channels/pe.m3u

```diff
@@ -136,6 +136,8 @@ https://7.innovatestream.pe:19360/omega/omega.m3u8
 https://ed1ov.live.opencaster.com/CwCfFGFdtebB/index.m3u8
 #EXTINF:-1 tvg-id="OndaDigital.pe" tvg-country="PE" tvg-language="" tvg-logo="https://i.imgur.com/3R2emx7.jpg" group-title="General",Onda Digital (720p) [Not 24/7]
 http://184.154.28.210:1935/ondadigital/ondadigital/playlist.m3u8
+#EXTINF:-1 tvg-id="OK TeVe.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/OKTelevision.pe/picture?width=320&height=320" group-title="Local",OK TeVe (Yurimaguas)
```

Comment:
As I understand it OK TeVe is the name of the service: http://okteve.com/, not the channel.

Reply:
It's not like you think. The website of this channel is https://oktelevisionhd.com/tv-en-vivo/.